sg/deps.go

```diff
@@ -47,7 +47,7 @@ func Deps(ctx context.Context, functions ...interface{}) {
 		go func() {
 			defer func() {
 				if v := recover(); v != nil {
-					errs[i] = fmt.Errorf(fmt.Sprint(v))
+					errs[i] = fmt.Errorf("%s", v)
```

Comment:
What's the context of this change?

Reply:
it fixes a govet lint
```
sg/deps.go:50:27: printf: non-constant format string in call to fmt.Errorf (govet)
errs[i] = fmt.Errorf(fmt.Sprint(v))
```
it was fixed by specifying a format for the value in fmt.Errorf